content/en/users/getting-started/cli/_index.md

```diff
@@ -1,11 +1,444 @@
+- Testing tools core features - Build Tools
+- Windows 10 SDK (`<latest`>)
+
+In case you prefer to use non-Microsoft alternatives for building non-pure
+packages, please see [here](https://wiki.python.org/moin/WindowsCompilers).
+
+To install the FedCloud client:
+
+```shell
+> pip3 install fedcloudclient
+```
+
+This installs the latest version of the FedCloud client, together with
+its required packages (like _openstackclient_). It will also create
+executables **fedcloud** and **openstack**, adding them to the _bin_
+folder corresponding to your current Python execution environment.
+
+{{< /tabx >}}
+{{< /tabpanex >}}
+
+Check if the installation is correct by executing the client:
+
+```shell
+$ fedcloud --version
+```
+
+#### Installing EGI Core Trust Anchor certificates
+
+Some sites in the EGI infrastructure use certificates issued by
+Certificate Authorities (CAs) that are not included in the default OS
+distribution. If you receive error message "_SSL exception connecting
+to..._", install the EGI Core Trust Anchor Certificates by running
+the following commands:
+
+<!-- markdownlint-disable line-length -->
+```shell
+$ wget https://raw.githubusercontent.com/tdviet/python-requests-bundle-certs/main/scripts/install_certs.sh
+$ bash install_certs.sh
+```
+<!-- markdownlint-enable line-length -->
+
+{{% alert title="Note" color="info" %}} The above script does not work on all
+Linux distributions. Change _python_ to _python3_ in the script if needed,
+see the [README](https://github.com/tdviet/python-requests-bundle-certs#usage)
+for more details, or follow the
+[official instructions](https://github.com/tdviet/python-requests-bundle-certs/blob/main/docs/Install_certificates.md)
+for installing EGI Core Trust Anchor certificates in production environments.
+{{% /alert %}}
+
+### Using via Docker container
+
+The FedCloud client can also be used without installation, by running it in a
+Docker container. In this case, the EGI Core Trust Anchor certificates are
+preinstalled.
+
+{{< tabpanex >}}
+{{< tabx header="Linux " >}}
+
+To run the FedCloud client in a container, make sure
+[Docker is installed](https://docs.docker.com/engine/install/#server),
+then run the following commands:
+
+```shell
+$ sudo docker pull tdviet/fedcloudclient
+$ sudo docker run -it  tdviet/fedcloudclient bash
+```
+
+{{< /tabx >}}
+{{< tabx header="Mac" >}}
+
+To run the FedCloud client in a container, make sure
+[Docker is installed](https://docs.docker.com/desktop/mac/install/),
+then run the following commands:
+
+```shell
+$ sudo docker pull tdviet/fedcloudclient
+$ sudo docker run -it  tdviet/fedcloudclient bash
+```
+
+{{< /tabx >}}
+{{< tabx header="Windows" >}}
+
+To run the FedCloud client in a container, make sure
+[Docker is installed](https://docs.docker.com/desktop/windows/install/),
+then run the following commands:
+
+```shell
+> docker pull tdviet/fedcloudclient
+> docker run -it  tdviet/fedcloudclient bash
```

Comment:
```suggestion
> docker run -it tdviet/fedcloudclient bash
```

I would probably replace `>` by `$` for consistency.

Reply:
Nope. On Windows that is the prompt. I will update the Copy script to accept > as the prompt too, not just $.

Reply:
You are right it makes sense.